packages/liveblocks-emails/src/index.ts

```diff
@@ -13,12 +13,30 @@ export type {
   ConvertCommentBodyAsHtmlStyles,
   ConvertCommentBodyAsReactComponents,
 } from "./comment-body";
+export type { ResolveRoomInfoArgs } from "./lib/types";
+export type {
+  ConvertLiveblocksTextEditorNodesAsReactComponents,
```

Comment:
When you have the word "convert", I think it should be "To" instead of "As." ConvertLiveblocksTextEditorNodesToReactComponents.

Also, these names are very long, could we maybe remove Liveblocks from them so they are a bit shorter?

Reply:
For `As` → `To` I will not rename it yet, I already deliver in prod for `ThreadNotificationEvent` with `As` in `ConvertCommentBodyAsReactComponents` or `ConvertCommentBodyAsHtmlStyles`.

But I will think about renaming to remove the `Liveblocks` part 👍🏻